app/models/persistent_volume.rb

```diff
@@ -0,0 +1,4 @@
+class PersistentVolume < ActiveRecord::Base
```

Comment:
@Fryguy did you mean ContainerPersistentVolume? Because if you consider this also for Cloud (and why not, also for Infra), then a generic `PersistentVolume` is better.